server/services/hospitalSearchService.test.ts

```diff
@@ -12,10 +13,8 @@ jest.mock('../data/prisonApiClient', () =>
 
 const user = {
   username: 'user1',
-  name: 'User',
-  activeCaseLoadId: 'MDI',
   token: 'token-1',
-} as User
+} as Context
```

Comment:
copied nomis sync dashboard approach of passing round `context` rather than the `user`